test/unit/Layouts.js

```diff
@@ -1,30 +1,10 @@
-/* global require, describe, d3, LocusZoom, beforeEach, afterEach, it */
-
 "use strict";
 
 /**
   layouts.js Tests
   Test composition of the LocusZoom.Layouts object
 */
-
-var jsdom = require("mocha-jsdom");
-var fs = require("fs");
-var assert = require("assert");
-var should = require("should");
-var files = require("../files.js");
-
 describe("Layouts", function(){
-
-    // Load all javascript files
-    var src = [];
-    files.test_include.forEach(function(file){ src.push(fs.readFileSync(file)); });
-    jsdom({ src: src });
-
-    // Reset DOM after each test
-    afterEach(function(){
-        d3.select("body").selectAll("*").remove();
-    });
```

Comment:
Really nice to see this redundancy abstracted away, thanks for doing that!

Reply:
...I was trying to address intermittent test failures (due to slow jsdom start) and got carried away. ;)

Now if I could just get sourcemaps working (via getting `gulp-wrap-js` to behave or otherwise), we'd have a pretty convenient developer setup for debugging all around...